pkg/networkpolicy/packet.go

```diff
@@ -29,59 +30,113 @@ func (p packet) String() string {
 // https://github.com/golang/net/blob/master/ipv4/header.go
 func parsePacket(b []byte) (packet, error) {
 	t := packet{}
-	if b == nil {
-		return t, fmt.Errorf("empty payload")
+	var errorTooShort = fmt.Errorf("Packet too short")
+	var errorCorrupted = fmt.Errorf("Packet corrupted")
```

Comment:
```suggestion
	var errorCorrupted = fmt.Errorf("packet corrupted")
```

ditto

Reply:
I think is better if you define the variables as global in the package so the consumer can cast the errors based on the error type